server/lib/notifications/agents/slack.ts

```diff
@@ -39,6 +39,7 @@ interface EmbedBlock {
 }
 
 interface SlackBlockEmbed {
+  text?: string;
```

Comment:
```suggestion
  text: string;
```
Since we always want to include `text` to avoid this issue in the future.

Reply:
Gotcha